ui/raidboss/data/06-ew/alliance/thaleia.txt

```diff
@@ -656,7 +658,6 @@ hideall "--sync--"
 5237.6 "Destructive Bolt" Ability { id: "8CEC", source: "Eulogia" }
 
 5300.0 label "Torrential Tridents"
-5297.8 "--middle--" Ability { id: "8A02", source: "Eulogia" }
 5300.0 "--sync--" StartsUsing { id: "8A4E", source: "Eulogia" }
```

Comment:
> Are these `"--middle--"`syncs actively incorrect? I might have missed the existing ones that would happen at the label jump points, but if I didn't this looks like a flat removal.

There's some discussion with @xiashtra about this above (thread marked as resolved, so probably didn't catch your eye).  I originally commented them out because they will never actually sync to anything (the `4400` lines all sync and jump to the labels that are 2.8 seconds after these), so they are basically just comments/context anyway.  We discussed whether to move them above the labels (it seemed weird to disable-sync-order for just one line), but per xiashtra:

> We could also drop them, especially since I notice they don't exactly match logs that get that far into the fight as it is (some of the re-centerings don't happen where indicated, and others that do happen are missing).

Reply:
Ah, my bad missing that. Carry on then.